spec/API_specification/data_type_functions.md

```diff
@@ -4,9 +4,72 @@
 
 A conforming implementation of the array API standard must provide and support the following data type functions.
 
+
 <!-- NOTE: please keep the constants in alphabetical order -->
 
 ## Objects in API
+(function-broadcast-arrays)=
+### broadcast_arrays(*args, /)
+
+Broadcast any number of arrays against each other.
+
+#### Parameters
+
+-   ***args**: _\[&lt;array&gt;\]_
+
+    -   arrays to broadcast.
+
+#### Returns
+
+-   **out**: _\[&lt;array&gt;\]_
+
+    -   a list of arrays where more than one element of a broadcasted array may refer to a single memory location.
+
+
+(function-broadcast-to)=
+### broadcast_to(array, /, shape)
```

Comment:
```suggestion
### broadcast_to(x, shape, /)
```

Was there a particular reason to make `shape` positional or keyword? We don't do this for the creation functions atm, and I am not convinced that the benefits of keyword support outweigh the need to support multiple ways of doing the same thing.

Reply:
Also, `x` is used as the argument convention for a single array argument.

Reply:
> Was there a particular reason to make shape positional or keyword?

Nope, this was my mistake 😅